mlir_graphblas/src/graphblas-opt/graphblas-opt.cpp

```diff
@@ -0,0 +1,36 @@
+//===- graphblas-opt.cpp ---------------------------------------*- C++ -*-===//
+//
+// TODO add doc string
+//
+//===---------------------------------------------------------------------===//
+
+#include "mlir/IR/Dialect.h"
+#include "mlir/IR/MLIRContext.h"
+#include "mlir/InitAllDialects.h"
+#include "mlir/InitAllPasses.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Pass/PassManager.h"
+#include "mlir/Support/FileUtilities.h"
+#include "mlir/Support/MlirOptMain.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/InitLLVM.h"
+#include "llvm/Support/SourceMgr.h"
+#include "llvm/Support/ToolOutputFile.h"
+
+#include "GraphBLAS/GraphBLASDialect.h"
+
+int main(int argc, char **argv) {
+  mlir::registerAllPasses();
+  // TODO: Register graphblas passes here.
+
+  mlir::DialectRegistry registry;
+  registry.insert<mlir::graphblas::GraphBLASDialect>();
+  registry.insert<mlir::StandardOpsDialect>();
+  // Add the following to include *all* MLIR Core dialects, or selectively
+  // include what you need like above. You only need to register dialects that
+  // will be *parsed* by the tool, not the one generated
+  // registerAllDialects(registry);
```

Comment:
Should we include all dialects?  I know we can chain this tool with the standard `mlir-opt` to get the rest of the dialects, but are there any downsides to an all-in-one tool?

Reply:
I don't see any downsides to creating an all-in-one tool.

This comment was left here as a result of using the [`standalone` MLIR template](https://github.com/llvm/llvm-project/blob/e7021232e66f4a8e21f2bcd77d9841d1fb414245/mlir/examples/standalone/standalone-opt/standalone-opt.cpp#L31). 

Perhaps it might be a good idea to decide on this when we write our first set of passes.